DesignSafe-CI/portal
# Support html apps

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [DES-2629](https://tacc-main.atlassian.net/browse/DES-2629)

## Testing Steps: ##
1. `npm ci` to install additional package
2. Add an html type app
3. Load the app in the portal
4. Confirm both html and tapis apps load normally

## UI Photos:
![Screenshot 2024-05-17 at 1 09 07 PM](https://github.com/DesignSafe-CI/portal/assets/20326896/0e58c6c3-e88a-4995-9049-825609bc0072)

## Notes

### Instructions to load local dev with prod data:
1. Follow step 10 here: https://tacc-main.atlassian.net/wiki/spaces/UP/pages/6659524/Local+Development+Environment+Setup
2. Run migrations
  a. first, `./manage.py migrate designsafe_auth --fake`
  b. then, `./manage.py migrate`
